### DISCUSSION TITLE
Multiple editors with one menu

### DISCUSSION THREAD

**Discussion by AndiLeni** [ueberdosis/tiptap#4495] [General Questions]
Hello,

is it somehow possible to connect multiple editors with just one menu? F.e. there are multiple editor areas on one page and all of them should react to a single sticky menu bar which is fixed to the top.

Thanks and kind regards

**Comment by DuyennTran** [comment]
Hello @AndiLenim, 

I also have the same question as you. I don't know if you have the answer yet, can you share it with me?

Thank you so much

**Reply by AndiLeni** [reply]
@DuyennTran no, unfortunately I don't know if this is possible, sorry. 
I didn't need this feature any more and hence didn't try again.
